robottelo/constants.py

```diff
@@ -2385,8 +2385,7 @@
     'virtualhw_version': 'Default',
     'pool': 'Resources',
     'network_interface_name': 'VMXNET 3',
-    'datastore': 'Local-Ironforge (free: 1.84 TB, prov: 1.14 TB, '
-                 'total: 2.72 TB)',
+    'datastore': 'Local-Ironforge (free: 1.66 TB, prov: 2.29 TB, total: 2.72 TB)',
```

Comment:
Do you put in constant file dynamic data like free space on a drive?

Reply:
I do not put that value in the constant (it was part of the original test), I only correct it with the current correct one. Otherwise we have to find a way to get that value and set it (that is not as logical as it appear), but we can think to implement in airgun the filtered dropdown to be able to select a value like 'Local-Ironforge* ' to filter by and than select the value that start with 'Local-Ironforge'.
What do you think?

Reply:
anyway, that solution will work for next 10 minutes as free space can change any moment. We can thing about improving current logic to be able to match exact value (common scenario) and by some template (for advanced cases)

Reply:
Resolved by dynamically recalculate that summary string from the summary data provided by the  vmware backend.